Dockerfile.ci

```diff
@@ -31,9 +31,29 @@ EOF
 
 WORKDIR /workspace
 
+# Install Mamba
+ARG MAMBA_TAG=v2.0.3
+ARG CAUSAL_CONV_TAG=v1.2.2.post1
+
+RUN <<"EOF" bash -ex
+# Mamba dependancy installation
+git clone --depth 1 --branch ${MAMBA_TAG} https://github.com/state-spaces/mamba.git && \
```

Comment:
Out of curiosity: why not a pip+git install instead? Are there benefits to this approach? No need to change it if results in the same

Reply:
I think it picks the package and we want to build it from source, I'm not sure how to trigger the same setup.py command via pip.